src/guided_fuzzing_daemon/utils.py

```diff
@@ -276,3 +276,22 @@ def close(self) -> None:
             if not self.hide:
                 open_file.print(flush=True)
             open_file.handle.close()
+
+
+def open_log_handle(pattern: str | None, tmp_base: Path, idx: int) -> TextIO:
```

Comment:
This code also exists in `afl.py`, and should be used there.

Reply:
Added in [d0a4b4c](https://github.com/MozillaSecurity/guided-fuzzing-daemon/pull/62/commits/d0a4b4c83875ed1bce301297a5df25b815d6781d).